.github/workflows/ci-windows.yml

```diff
@@ -68,3 +68,48 @@ jobs:
           chdman.exe
           unidasm.exe
         if-no-files-found: error
+
+  build-windows-arm64:
+    runs-on: windows-11-arm
+    defaults:
+      run:
+        shell: msys2 {0}
+    strategy:
+      matrix:
+        subtarget: [mame, tiny]
+        include:
+          - subtarget: mame
+            executable: mame
+          - subtarget: tiny
+            executable: mametiny
+    steps:
+    - uses: msys2/setup-msys2@v2
+      with:
+        msystem: CLANGARM64
+        install: git make mingw-w64-clang-aarch64-cc mingw-w64-clang-aarch64-python mingw-w64-clang-aarch64-libc++ mingw-w64-clang-aarch64-gcc-compat
+    - uses: actions/checkout@main
+      with:
+        fetch-depth: 0
+    - name: Build
+      env:
+        MINGW64: "/clangarm64"
+        OVERRIDE_AR: "llvm-ar"
+        OVERRIDE_CC: clang
+        OVERRIDE_CXX: clang++
+        ARCHOPTS: "-fuse-ld=lld"
+        SUBTARGET: ${{ matrix.subtarget }}
+        ARCHOPTS_C: " -Wno-nontrivial-memcall -march=armv8.2-a"
+        ARCHOPTS_CXX: " -Wno-nontrivial-memcall -march=armv8.2-a"
```

Comment:
What’s triggering the `nontrivial-memcall` warning?  We don’t do these kinds of workarounds in CI files:
* In general we try to address warnings if it makes sense and the compiler isn’t just being stupid.
* We demote warning-as-error to warning in the Lua build scripts when it’s something we want to clean up.
* We suppress problematic warnings that we can’t work around in the Lua build scripts for third-party libraries.

Why do you need to specify `-march` options?  What does it default to?

Reply:
- I can remove  `-Wno-nontrivial-memcall` because only Clang 20 will fail to build it, but the mamedev runners are using Clang 19 still which I think will be fine.
 
- `-march` is necessary to bring the instruction set that 7z is expecting. 8.2 is totally fine because even the Snapdragon 8cx (the one from 2019) supports it.

Reply:
I’d still rather leave it with the default profile for now.  We aren’t supplying `-march` for x86-64 builds either, even though our binary releases require x86-64-v2 features.